src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -158,7 +158,7 @@ repository:
 
   implicit-expression:
     name: 'meta.expression.implicit.cshtml'
-    begin: '(@)'
+    begin: '(?<![[:alpha:][:alnum:]])(@)'
```

Comment:
This isn't technically a complete email regex, but I assume we're going for a "best effort, highly readable" version until there's a compelling reason to do more complicated stuff?

Reply:
Ya we're trying to match what's done in the Razor parser. It's not complete but it's consistent 😄